src/views/RecIM/views/Match/index.jsx

```diff
@@ -89,6 +95,14 @@ const Match = () => {
     setOpenMatchForm(false);
   };
 
+  const handleDelete = () => {
+    deleteMatchCascade(matchID);
+    setOpenConfirmDelete(false);
+    setOpenSettings(false);
+    navigate(`/recim/activity/${match.Activity.ID}`);
```

Comment:
Hypothetically we could start pre-emptively sending data back to our pages for snackbar information
i.e
```
let res = deleteMatchCascade(matchID); //pretty sure this returns something or at least it should
. . .
navigate(`/recim/activity/${match.Activity.ID}`, {"match" : res}); //pass the prop back. I don't think it'll break anything
```